packages/kotti-ui/source/kotti-popover/KtPopover.vue

```diff
@@ -19,13 +26,27 @@
 import { createPopper } from '@popperjs/core'
 import { mixin as clickaway } from 'vue-clickaway'
 
+import KtPopoverIconTextItem from './KtPopoverIconTextItem.vue'
+
+const optionIsValid = (option) =>
```

Comment:
so, let's tackle this line for line

what if your `option` is a `null` - what if you're given an array of `null` or `undefined` - this line for starters will fail
so you need to above anything guard against `option` being `undefined`/ `null`, 
then what if your option has a `string` type, for example .. well you also need to guard then that this option is an object, no?
Your first check should be
> `typeof option === 'object'` && object !== null`
the `object` `typeof` check will guard against all wrong types and the the second part is immediately augemented because `null` passes as object